src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import org.broadinstitute.dsde.workbench.leonardo.model._
+
+// values common to multiple tests, to reduce boilerplate
+
+trait CommonTestData {
```

Comment:
Yay! Should we put cluster[1-3] here too, as they seem to be copy-pasted everywhere?

Reply:
E.g. 
```
val c1 = Cluster(
      clusterName = "name1",
      googleId = UUID.randomUUID(),
      googleProject = "dsp-leo-test",
      googleServiceAccount = "not-a-service-acct@google.com",
      googleBucket = "bucket1",
      clusterUrl = Cluster.getClusterUrl("dsp-leo-test", "name1"),
      operationName = "op1",
      status = ClusterStatus.Unknown,
      hostIp = Some("numbers.and.dots"),
      createdDate = Instant.now(),
      destroyedDate = None,
      labels = Map("bam" -> "yes", "vcf" -> "no"),
      jupyterExtensionUri = None)
```

Reply:
sure!

Reply:
actually no - they have diverged too much